src/map/status.cpp

```diff
@@ -2351,8 +2350,13 @@ int32 status_base_amotion_pc(map_session_data* sd, struct status_data* status)
 		val -= 50 - 10 * pc_checkskill(sd, KN_CAVALIERMASTERY);
 	else if (pc_isridingdragon(sd))
 		val -= 25 - 5 * pc_checkskill(sd, RK_DRAGONTRAINING);
-	amotion = ((int32)(temp_aspd + ((float)(status_calc_aspd(&sd->bl, &sd->sc, true) + val) * status->agi / 200)) - min(amotion, 200));
+	aspd = ((int32)(temp_aspd + ((float)(status_calc_aspd(&sd->bl, &sd->sc, true) + val) * status->agi / 200)) - min(aspd, 200));
+	return aspd;
 #else
+	if (job == nullptr)
+		return ZERO_ASPD/AMOTION_DIVIDER_PC;
+
+	int32 amotion;
 	// Angra Manyu disregards aspd_base and similar
 	if (pc_checkequip2(sd, ITEMID_ANGRA_MANYU, EQI_ACC_L, EQI_MAX))
 		return 0;
```

Comment:
Shouldnt this return MAX_ASPD for the specific job?

Note: If we can dehardcode this, that would also be good.

Reply:
This is just some custom feature. You can accomplish the same thing by giving the item a high ASPD bonus. We could just remove it? Maybe a separate PR. Just fixing the two GM items for both pre-re and re.

Reply:
Ok fine for me!